### ISSUE TITLE
disable annoyance

### ISSUE THREAD

**Issue by ratpackbn** [MTACS/Lynx#182]
when you go into airplane mode, and the wifi is also off, u get that annoying alert to turn off airplane mode. please make a switch to disable that. thanx!

**Comment by NewGithubber35** [comment]
Yes, agreed. Good suggestion. I use ethernet cable to hard wire my iPad and this annoying notification comes up every time even though I'm already connected to the internet.
